.vscode/settings.json

```diff
@@ -1,3 +1,18 @@
 {
-  "typescript.tsdk": "node_modules/typescript/lib"
+  "typescript.tsdk": "node_modules/typescript/lib",
+  "debug.javascript.terminalOptions": {
```

Comment:
I have added this issue : https://github.com/input-output-hk/marlowe-ts-sdk/issues/162 , because I'm not able to attach the debugger ti VSCode since we moved to packages